os-config_arm.yml

```diff
@@ -0,0 +1,328 @@
+hostname: rancher
```

Comment:
Maybe tag images `rancher/os-xxx-ARCH:VERSION` and replace `ARCH` and `VERSION` via a `sed` command before using the file? That would solve the problem...